package.json

```diff
@@ -20,44 +20,44 @@
 			"go"
 		]
 	},
-	"devDependencies": {
-		"@defense-unicorns/unicorn-ui": "^0.0.26",
+	"dependencies": {
+		"@defense-unicorns/unicorn-ui": "0.0.27",
 		"@fontsource/roboto": "4.5.8",
 		"@material/card": "14.0.0",
 		"@material/data-table": "14.0.0",
 		"@material/layout-grid": "14.0.0",
 		"@material/linear-progress": "14.0.0",
 		"@material/theme": "14.0.0",
-		"@playwright/test": "1.28.1",
-		"@sveltejs/adapter-static": "next",
-		"@sveltejs/kit": "next",
-		"@sveltejs/package": "next",
+		"material-symbols": "0.4.2",
+		"prismjs": "1.29.0",
+		"sanitize.css": "13.0.0",
+		"sass": "1.57.1",
```

Comment:
Why is sass a dependency now?

Reply:
I find the whole dep vs devdep weird with svelte tbh in how it is setup. They difference seems to be meaningless.

Reply:
Yeah I think most stuff should be devDeps basically right?

Reply:
Yeah think so, @Noxsios  or @mike-winberry may have thoughts.

Reply:
It is 100% subjective, sometimes a dep prefers to be in one or the other. My logic is that things that help the build process (like typescript and any `@type`, etc...) go in devDep, vs deps that can be seen in the UI (ie material, prism) go in deps.  sass is a weird one where in my mind it could go in either.

Reply:
Sass is being used as a css pre-processor though right?  If so it would not be in the UI since that would only see regular CSS.  My personal opinion on deps vs devDeps is that if it is code that runs in the browser then it is a dep, but if it is code that generates code from other code that runs in the browser it is a devDep.  From what I know, in this case typescript and sass are doing the same thing

Reply:
yeah I'd lean towards deps are things that are needed at runtime (server or browser).